data.json

```diff
@@ -520,5 +520,11 @@
     "githubUsername": "webdevrachit",
     "imageUrl": "https://user-images.githubusercontent.com/72686780/106394124-fa89d480-6420-11eb-837b-fbf179a7697d.png",
     "issueNumber": 286
+  },
+  {
+    "name": "Varghese Jose",
+    "githubUsername": "varghesejose2020",
+    "imageUrl": "https://user-images.githubusercontent.com/71702982/106555821-526d2c00-6544-11eb-9da0-076566844959.JPG",
+    "issueNumber": 287
```

Comment:
What is the issue number? #287 is a Pull Request.

Reply:
What to fill in that portion

Reply:
issue number updated